test/mpi/coll/test_coll_algos.sh

```diff
@@ -262,4 +262,23 @@ for algo_name in ${algo_names}; do
     done
 done
 
+######### Add tests for Gatherv algorithms ###########
+
+#disable device collectives for gatherv to test MPIR algorithms
+testing_env="env=MPIR_CVAR_GATHERV_DEVICE_COLLECTIVE=0 "
+
+#test nb algorithms
+testing_env+="env=MPIR_CVAR_GATHERV_INTRA_ALGORITHM=nb "
+testing_env+="env=MPIR_CVAR_IGATHERV_DEVICE_COLLECTIVE=0 "
+algo_names="gentran_linear"
+
+for algo_name in ${algo_names}; do
+    for kval in ${kvalues}; do
+        #set the environment
+        env="${testing_env} env=MPIR_CVAR_IGATHERV_INTRA_ALGORITHM=${algo_name} "
+
+        coll_algo_tests+="gatherv 5 ${env}${nl}"
+    done
```

Comment:
I could not find the test `gatherv.c`, where was this added?

Reply:
You're right there isn't any test called `gatherv.c` and AFAIK it wasn't added either. I will update the test here. I didn't check it when I created the PR for Akhil's commits.

Reply:
So there was a commit that renamed `coll3` test to `gatherv` and I missed that. I added that commit now.